### DISCUSSION TITLE
Can I integrate the template that I bought from ThemeForest with backpackforlaravel?

### DISCUSSION THREAD

**Discussion by christmex** [Laravel-Backpack/community-forum#351] [General] [answered]
Hi guys, is anyone there already integrated a custom template with backpackforlaravel? 

Thank you.

**Comment by tabacitu** [comment, marked as answer]
In short:
- In Backpack v5, yes you can, but it's not advised; [how to here](https://backpackforlaravel.com/docs/5.x/base-how-to#create-a-new-theme-child-theme);
- In Backpack v6 (coming March 2023) it'll be an official feature, non-breaking and well documented;

The difference between the two systems will be that in Backpack v6 we'll be splitting up the views - each view will only have one responsibility, so when you need to change something, you'll only be overriding a small bit.

**Reply by christmex** [reply]
Thank you so much, will wait the new versionnn
